src/app/shared/documentation-items/documentation-items.ts

```diff
@@ -29,7 +29,7 @@ const DOCS = [
   },
   {
     id: 'nav',
-    name: 'Navigation',
+    name: 'Navigation Components',
```

Comment:
It's somewhat redundant for this to say "Navigation components" since they're all components. Could it perhaps just be done as an aria label?